src/django/pfb_analysis/models.py

```diff
@@ -142,26 +190,22 @@ def create_environment(**kwargs):
                                      containerOverrides=container_overrides)
         try:
             self.batch_job_id = response['jobId']
-            # TODO: Possibly refactor status to use the AWS Batch status rather than some
-            #       custom status we may not have the ability to easily update
-            self.status = self.Status.IMPORTING
             self.save()
         except (botocore.exceptions.BotoCoreError, KeyError):
             logger.exception('Error starting AnalysisJob {}'.format(self.uuid))
 
-    def batch_job_status(self):
-        """ Return current AWS Batch job status for this job
 
-        List of available statuses: http://docs.aws.amazon.com/batch/latest/userguide/jobs.html
-        TODO: Refactor to cache in db?
+class AnalysisJobStatusUpdate(models.Model):
+    job = models.ForeignKey(AnalysisJob, related_name='status_updates', on_delete=models.CASCADE,
```

Comment:
Is there a case where we would want to create an instance of this model but not have a job to attach it to? e.g. should we be using null=True and blank=True here?